templates/master/00-master/two-node-with-fencing/extensions/two-node-ha

```diff
@@ -0,0 +1,2 @@
+# The content of this file is ignored.
+ # Extensions are enabled based on the names of the files in the extensions dir
```

Comment:
I know this is my fault, but can we remove the random leading space in this file. :see_no_evil: